doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1001,90 +911,61 @@ Induction
 
 .. seealso:: :tacn:`functional inversion`
 
-.. todo move the following label to just
-        before ":tacn:`discriminate` and :tacn:`injection`"
-
-.. _equality-inductive_types:
-
-Equality and inductive sets
----------------------------
-
-We describe in this section some special purpose tactics dealing with
-equality and inductive sets or types. These tactics use the
-equality :g:`eq:forall (A:Type), A->A->Prop`, simply written with the infix
-symbol :g:`=`.
-
 .. tacn:: decide equality
```

Comment:
I guess there should be a sub-section title here. Something like "Helper tactics". Maybe @JasonGross would have a better suggestion.

Another possibility is to move the various helper tactics closer to the tactic they are useful for.

Reply:
I'd lean toward moving the "helper tactics" closer to the the ones they help.  "Helper tactics" is not a very informative title.  The helper tactics are all of `decide equality` .. `dependent rewrite`?  Where should each one go?

Reply:
Correct. Most of them already mention in their description which tactic they are useful for. The other ones, we would have to ask.